firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/metadata/RolloutAssignment.java

```diff
@@ -58,7 +65,13 @@ static RolloutAssignment create(String json) throws JSONException {
     String parameterValue = dataObj.getString("parameterValue");
     String variantId = dataObj.getString("variantId");
     long templateVersion = dataObj.getLong("templateVersion");
+
+    String validatedParameterValue = parameterValue;
+    if (validatedParameterValue.length() > MAX_PARAMETER_VALUE_LENGTH) {
+      validatedParameterValue = validatedParameterValue.substring(0, MAX_PARAMETER_VALUE_LENGTH);
+    }
```

Comment:
Being a little nitpicky - could we put the validation in a method? So we can say `String validatedParameterValue = validate(parameterValie)` both here and up at line 51?

Reply:
Done!